src/index.js

```diff
@@ -1250,8 +1250,18 @@ module.exports = {
    * @param {buffer} privateKey The sender's plaintext private key.
    * @return {string} Signed and serialized raw transaction.
    */
-  signRawTransaction: function (packaged, privateKey) {
-    var rawTransaction = new EthTx(packaged);
+  signRawTransaction: function (packaged, privateKeyOrSigner, callback) {
+    try {
+      var transaction = new EthTx(packaged);
+      return (typeof privateKeyOrSigner === "function")
+        ? privateKeyOrSigner(transaction, callback)
+        : this.signTransactionWithKey(transaction, privateKeyOrSigner, callback);
+    } catch (error) {
+      return callback(error, undefined);
```

Comment:
While I personally like `callback(error, result)`, I recommend using the `callback(error || result)` style here for consistency.